.github/workflows/create-stable-version.yml

```diff
@@ -33,16 +33,6 @@ jobs:
           git checkout -b ${{ github.event.inputs.version }} origin/unstable
           git push -u origin ${{ github.event.inputs.version }}
 
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v5
-        with:
-          token: ${{ secrets.SHOPIFY_GH_ACCESS_TOKEN }}
-          commit-message: 'Add ${{ github.event.inputs.version }} to watched branches'
-          title: 'Add ${{ github.event.inputs.version }} to watched branches'
-          body: 'This PR adds the new stable version ${{ github.event.inputs.version }} to the list of watched branches in the changesets-reminder and deploy GitHub actions.'
-          branch: 'update-watched-branches-${{ github.event.inputs.version }}'
-          base: '${{ github.event.inputs.version }}'
-
```

Comment:
You are removing the PR that gets created, but you still create the branch with the updated GH action files? I'm not sure if I understand correctly.

Reply:
Ahh I see what you mean. Yes good call, we don't need that either.